scripts/install_dependencies.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+pip install ../../backend/sentiment/requirements.txt
```

Comment:
this is wrong, it should be

pip install -r ../backend/requirements.txt

-r is needed anyway for requirements files but you also put the wrong directory here.

Reply:
Thanks for reminding

Reply:
I was really asking if #!/usr/bin/env bash is correct